scripts/src/custard.test.ts

```diff
@@ -368,53 +368,48 @@ describe('listSecrets', () => {
 });
 
 describe('run', () => {
-  it('undefined commands', () => {
```

Comment:
This tests is removed because validation is now done in the main function when we validate CLI arguments, so `custard.run` always gets a valid command.